### PR TITLE
Create J Boss.html

### PR DESCRIPTION
**Pull request by Guneet06**
It was easy

### CODE REVIEW

**Comment by GuneetSasan** [comment]
I am not getting it

**Comment by GuneetSasan** [comment]
What do you mean

File: J Boss.html [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+<body>
+	<style>
+.how
+{
+	height: 100vh;
+	background-size: cover;
+	background-position: center;
+}
+.hell{
+float:center;
```
Comment (nikitadhiman) [inline J Boss.html]: You should never add css in html files. In angular we have <component-name>.component.css and hence you should make your css changes in the same file. For all frontend projects we are using angular and hence follow the guidelines 
for the same.

**Comment by nikitadhiman** [comment]
closing this as no further changes made

**Pull request closed without merging**